CREDITS

```diff
@@ -159,7 +164,8 @@ of the Coq Proof assistant during the indicated time:
   Amin Timany (Katholieke Universiteit Leuven, 2017)
   Benjamin Werner (INRIA, 1989-1994)
   Nickolai Zeldovich (MIT 2014-2016)
-  Théo Zimmermann (INRIA-IRIF, 2015-now)
+  Théo Zimmermann (ORCID: https://orcid.org/0000-0002-3580-8806,
```

Comment:
My name appears a few lines above. For that entry, you should change "now" to "2018". :-(

Reply:
OK :-(